network-transport.cabal

```
@@ -68,7 +68,7 @@ Library
                    binary >= 0.5 && < 0.9,
                    bytestring >= 0.9 && < 0.12,
                    hashable >= 1.2.0.5 && < 1.5,
-                   transformers >= 0.2 && < 0.6,
+                   transformers >= 0.6 && < 0.7,
```

Thanks @epoberezkin! Is the update to the lower bound necessary?
```suggestion
                   transformers >= 0.2 && < 0.7,
```